examples/posenet/index.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html lang = "en">
+  <head> 
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title> PoseNet Image Demo</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <style>
+      #gui{
+        position: relative;
+        top: 100px;
+      }
+      .main{
+        position: absolute;
+        left: 50px;
+        top: 100px;
+        right: 0px;
+      }
+      .col{
+        position: relative;
+        top: 20px;
+      }
+      .Single,.Multiple {
+        display: inline-block;
+      }
+    </style>
+  </head>
+
+  <body>
+    <nav class="navbar navbar-expand-md navbar-dark bg-dark" style="width:100%">
+      <span class="navbar-brand mb-0 h1">WebML PoseNet Demo</span>
+        <div class="btn-group">
+          <select onchange = "drawSingleandMulti();" id = "backend" class="btn btn-success dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <option value = "webml">WebML</option>
+            <option value = "wasm">WASM</option>
+            <option selected = "selected">WebGL</option>
+          </select>
+        </div>
+    </nav>
+
+    <div id = "loading">
+      Loading the model...
+    </div>
+    <div class = "main" id = "main">
+      <div class = "Single">
+        <h4>Single Pose Estimation</h4>
+        <canvas id = "canvas" width = "513" height = "513"></canvas>
+      </div>
+      <div class = "Multiple">
+        <h4>Multiple Poses Estimation</h4>
+        <canvas id = "canvas_2" width = "513" height = "513"></canvas>
+      </div>
+
+      <div class ="col" align="center">
+        <label id="button" class="btn btn-primary disabled">
+          Pick Image <input hidden onchange = "drawResult();" type="file" id="image" accept="image/">
+        </label>
+      </div>
+    </div>
+    <canvas hidden id = "scaleImage"></canvas>
```

Comment:
Please add `inference time` to the UI. Please refer to https://github.com/intel/webml-polyfill/blob/master/examples/mobilenet/index.html#L77. Same for camera.html.